usl_models/tests/flood_ml/flood_model_test.py

```diff
@@ -262,3 +157,24 @@ def test_model_checkpoint():
     new_weights = new_model._model.get_weights()
     for old, new in zip(old_weights, new_weights):
         np.testing.assert_array_equal(old, new)
+
+
+def test_get_temporal_window():
+    """Tests copmuting the temporal window."""
+    B, T_MAX, M = 1, 8, 6
+    temporal = tf.constant([[[t] * M for t in range(T_MAX)]], dtype=tf.float32)
+
+    t = 3  # Timestep
+    n = 5  # Window size
+    actual = flood_model.FloodConvLSTM._get_temporal_window(temporal, t=t, n=n)
+    assert actual.shape == (B, n, M)
+    expected = [
+        [
+            np.zeros((M)),
+            np.zeros((M)),
+            np.zeros((M)),
+            [1, 1, 1, 1, 1, 1],
+            [2, 2, 2, 2, 2, 2],
+        ],
```

Comment:
Checking my understanding here. With `t = 3`, we have 2 rows of zero padding, and the data retrieved from `temporal` are rows 0-2, correct?

Could we set temporal to 1-8 instead? That row of zeros might be confusing.